<launch>
    <arg name="save_name" default="test"/>
    <node name="streamer" pkg="blimp_navigation" type="stream_file_node" output="screen" required="true">
        <param name="constant_rate" value="true"/>
        <param name="left_video" value="/home/veerachart/Videos/Left_20180608-205121.mp4"/>
        <param name="right_video" value="/home/veerachart/Videos/Right_20180608-205123.mp4"/>
        <param name="left_start_ms" value="13700"/>
        <param name="right_start_ms" value="11300"/>
    </node>
    
    <node name="track_left" pkg="blimp_navigation" type="blimp_human_track_node" output="screen">
        <remap from="camera" to="left"/>
        <param name="detect_human" value="true" />
        <param name="file_name" command="date +'$(find blimp_navigation)/log/%m%d_%H%M-$(arg save_name)_l'"/>
    </node>
    
    <node name="track_right" pkg="blimp_navigation" type="blimp_human_track_node" output="screen">
        <remap from="camera" to="right"/>
        <param name="detect_human" value="true" />
        <param name="file_name" command="date +'$(find blimp_navigation)/log/%m%d_%H%M-$(arg save_name)_r'"/>
    </node>
    
    <node name="triangulator" pkg="blimp_navigation" type="triangulate_blimp_node" output="screen"/>
    
    <node name="viewer_left" pkg="image_view" type="image_view">
        <remap from="image" to ="/cam_left/detection_image"/>
    </node>
    
    <node name="viewer_right" pkg="image_view" type="image_view">
        <remap from="image" to ="/cam_right/detection_image"/>
    </node>
</launch>
